### PR TITLE
#183 Insert mock data for assignment, reviews and consolidations

### LINKED ISSUE #183: Add new Reviews for ReviewTest application with different status

### PR DESCRIPTION
**Pull request by nmadruga**
Fix #183 

### Summary
- Add levels to each stage of `ReviewTesting` as described in the [comment#188](https://github.com/openmsupply/application-manager-server/issues/188#issuecomment-772016513) 
- Add 3 submitted applications for `template_D_basicReviewTesting` with ids: 4000, 4001, 4002. 
- And for each application submitted, reviews in different stages/statuses:

### 1. Awaiting Stage 2 lvl 2 consolidation to start:
  - Stage 1: Approved (Reviewer 1)
  - Stage 2: (Level 1 - 2 reviewers)
    - Reviewer 1: submitted Approved
    - Reviewer 2: submitted Denied
  - Stage 2 (Level 2 - 2 consolidators)
    - Consolidator 1: not started
    - Consolidator 2: not started
### 2. Awaiting Stage 1 to start:
  - Reviewer 1 - Assigned but not started
### 3. Awaiting Stage 2 lvl 1 to continue (changes requested):
   - Stage 1: Approved (Reviewer 1)
  - Stage 2: (Level 1 - 2 reviewers)
    - Reviewer 1: submitted Denied - Changes requested
    - Reviewer 2: Not started
  - Stage 2 (Level 2 - 2 consolidators)
    - Consolidator 1: Not available -> Consolidator 2 start first
    - Consolidator 2: Submitted (Disagreement with Reviewer 1)

### CODE REVIEW

File: database/insertData/04_users.js [hunk 1]
```diff
@@ -99,7 +99,35 @@ exports.queries = [
...
+      user {
+        username
+      }
+    }
+  }`,  
+  `mutation {
+    createUser(
+      input: {
+        user: { email: "consolidator1@sussol.net"
+          passwordHash: "$2a$10$pzhH6GcC7rw38AencBcbCuDaN6ANGZnVnE3ViCa5veOeTelbkkkv2"
```
Comment (CarlosNZ) [inline database/insertData/04_users.js:114]: Strictly speaking they should all have unique hashes. Not that it really matters for testing purposes, but if you can be bothered: https://www.browserling.com/tools/bcrypt

**Review by CarlosNZ — APPROVED** [review]
Awesome, thanks heaps for sorting this.

All looks good in the database, will be very useful for testing during review/assign development.

File: database/insertData/07_review_assignments.js [hunk 2]
```diff
@@ -0,0 +1,820 @@
...
+              { id: 1006, templateElementId: 4008 }
+              { id: 1007, templateElementId: 4009 }
+              { id: 1008, templateElementId: 4011 }
+              { id: 1009, templateElementId: 4012 }
+              { id: 1010, templateElementId: 4013 }
+            ]
+          }
+          reviewsUsingId: {
+            create: {
+              level: 1
```
Comment (CarlosNZ) [inline database/insertData/07_review_assignments.js:36]: Sorry, Nicole -- there is one thing I just noticed. The "level" value should be set on the "review_assignment" table, not the review table. The one in the review_table will be auto-generated from the review_assignment one.

**Review by CarlosNZ — CHANGES_REQUESTED** [review]
Sorry, I already approved, but then I spotted this. See my comment, should be a quick fix.

File: database/insertData/07_review_assignments.js (hunk 2 above)
Comment (nmadruga) [reply database/insertData/07_review_assignments.js]: Done : )

**Review by CarlosNZ — APPROVED** [review]
Brilliant, thank you :)

File: database/insertData/04_users.js (hunk 1 above)
Comment (nmadruga) [reply database/insertData/04_users.js:114]: Ah, nah. For now we want them to all have the same, so we can easily change from one to another

**Pull request merged**